pkg/kubernetes/client/context.go

```diff
@@ -53,29 +48,33 @@ func writeNamespacePatch(context objx.Map, namespace string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	if err = ioutil.WriteFile(f.Name(), []byte(out), 0644); err != nil {
+
+	defers.Defer(func() {
+		os.RemoveAll(f.Name())
```

Comment:
Doing this in a global function feels a little awkward to me. Personally, I'd want a cleanup or `Close` function on the Kubernetes object that deletes the nsPatch file. This is fine for now, but just food for thought.